testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksChangeDataFeedCompatibility.java

```diff
@@ -59,7 +60,7 @@ public void testUpdateTableWithCdf()
                             row("testValue4", 4, "update_postimage", 5L));
         }
         finally {
-            onDelta().executeQuery("DROP TABLE IF EXISTS default." + tableName);
+            onTrino().executeQuery("DROP TABLE IF EXISTS delta.default." + tableName);
```

Comment:
Can we add simple tests that verify change data feed isn't generated in the two cases? 
1. Without `change_data_feed_enabled` table properties
2. With `change_data_feed_enabled` false

Reply:
Sure I will, it is a pity I have to use compatibility tests to verify this. 
I am working on a table_changes function and there I can easily test all possibilities in integration tests. 
For now however I will probably revert the changes I made to that one particular test and write a dedicated test.